openedx/core/djangoapps/oauth_dispatch/management/commands/tests/test_clear_expired_tokens.py

```diff
@@ -46,8 +48,37 @@ def test_invalid_expiration_time(self):
                     )
                 )
 
-    @patch('oauth2_provider.settings.oauth2_settings.REFRESH_TOKEN_EXPIRE_SECONDS', 3600)
+    @override_settings()
+    def test_excluded_application_ids(self):
+        settings.OAUTH2_PROVIDER['REFRESH_TOKEN_EXPIRE_SECONDS'] = 3600
```

Comment:
I was under the impression from our HipChat conversation that this _does not_ work as expected. Is the value actually be set to 3600, and only broken when set to `None`?

Reply:
yes

Reply:
yeah for those its set to 3600 but even we set the value to None for one it keeps it as 3600 like other tests